docs/changehistory/NextVersion.md

```diff
@@ -3,6 +3,108 @@ publish: false
 ---
 # NextVersion
 
+Overview:
+
+- Breaking Changes:
+  - [Breaking API changes](#breaking-api-changes)
+  - [BriefcaseManager changes](#briefcasemanager-breaking-changes)
+- Visualization
+  - [Section-cut graphics](#section-cut-graphics)
+  - [Custom screen-space effects](#custom-screen-space-effects)
+  - [Automatic viewport synchronization](#automatic-viewport-synchronization)
+  - [Globe location tool fixes](#globe-location-tool-fixes)
+  - [Changes to display style excluded elements](#changes-to-display-style-excluded-elements)
+
+- [Tile compression](#tile-compression)
```

Comment:
You can probably shove this under "Visualization".

Reply:
Updated. I went back and forth on that one initially so sounds good to me.